<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zhuifan</class>
 <widget class="QDialog" name="zhuifan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>追番</string>
  </property>
  <widget class="QTextBrowser" name="weekindex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>651</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sun">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>星期日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>星期一</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>星期二</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wed">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>星期三</string>
   </property>
  </widget>
  <widget class="QPushButton" name="thu">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>星期四</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fri">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>星期五</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sat">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>星期六</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="saveindex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>651</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已收藏：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchtext">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchbtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="index01">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index02">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index03">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index04">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index05">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index06">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index07">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index08">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index14">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index17">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index09">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index15">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="index18">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>250</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="first">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Bilibili</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Soku</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="save01">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save02">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save03">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save05">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save06">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save04">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save08">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save09">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save07">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>490</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="save10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
